tests/integration/components/side-test.js

```diff
@@ -13,7 +13,7 @@ test('tooltip-on-element shows on the top by default', function(assert) {
 
   assert.expect(1);
 
-  this.render(hbs`{{tooltip-on-element keepInWindow=false}}`);
+  this.render(hbs`{{tooltip-on-element keepInWindow=false effect='none'}}`);
```

Comment:
Added because the `effect` class was causing spacing to be incorrect. The default `'fade'` effect moves about `10px` closer to the target prior to the tooltip being animated when the tooltip is shown.